src/vconsole/vconsole-setup.c

```diff
@@ -211,7 +211,7 @@ static void font_copy_to_all_vcs(int fd) {
 
         for (i = 1; i <= 15; i++) {
                 char vcname[strlen("/dev/vcs") + DECIMAL_STR_MAX(int)];
-                _cleanup_close_ int vcfd = -1;
+                int vcfd;
```

Comment:
Cleanup attribute is that clever ? I never used it myself, but I thought it triggers only when going out of scope.